app/scripts/components/common/map/hooks/use-maps.ts

```diff
@@ -1,13 +1,19 @@
 import { useContext } from 'react';
 import { useMap } from 'react-map-gl';
 import { MapsContext } from '../maps';
+import { StylesContext } from '../styles';
 
 export default function useMaps() {
   const { mainId, comparedId } = useContext(MapsContext);
+  const { isCompared } = useContext(StylesContext);
```

Comment:
nit: Do we want yo use context selectors (like the `useMapStyle`) or not worth it?